### ISSUE TITLE
deepvariant for low-deep sequencing?

### ISSUE THREAD

**Issue by LvLH** [google/deepvariant#107]
Hi, could you tell me if deepvariant can used for whole-genome low-deep sequencing (about 0.8X)? Thank you.

**Comment by pichuan** [comment]
Since DeepVariant isn't trained on this data, it will depend on how different this new datatype is from our existing training data. For the training data we used, you can see: https://github.com/google/deepvariant/blob/r0.7/docs/deepvariant-details-training-data.md

DeepVariant is known to be robust even on certain new data types. For example, when DNAnexus published this blog post, DeepVariant wasn't trained on NovaSeq, and yet DeepVariant performed well on NovaSeq data:
https://blog.dnanexus.com/2018-01-16-evaluating-the-performance-of-ngs-pipelines-on-noisy-wgs-data/

However, we also know that DeepVariant can be significantly more accurate if it has a chance to see the data in its training set. (For example, see: https://ai.googleblog.com/2018/04/deepvariant-accuracy-improvements-for.html)

If you'd like to give it a try and let us know, that will be great, especially if you compare it to other baselines. Thanks!

**Comment by pichuan** [comment]
I'll close this issue if there's no follow up actions. Feel free to re-open if you have more questions.
